routing/router.go

```diff
@@ -17,7 +17,7 @@ func NewRouter() *mux.Router {
 	webRoot := conf.Server.WebRoot
 	webRootWithSlash := webRoot + "/"
 
-	rootRouter := mux.NewRouter().StrictSlash(false)
```

Comment:
This was changed as part of the custom context root work (https://github.com/kiali/kiali/commit/3629100a3ca78a6c3831305c2086f32ba00e9998)

Are we sure this continues to work when the context root is something else?

Reply:
@mwringe 
I tested on https://github.com/kiali/kiali-client-python/commit/3e3c06dc1c5ed2eaeb6f19d296d4b6cb4de98326 and it worked with custom context